test/unit/spriteManipulation.js

```diff
@@ -385,3 +385,12 @@ test('setPropRandom set sprite y properties between 50 and 350', async t => {
 
   nativeAPI.reset();
 });
+
+test('p5 fixedSpriteAnimationSizes is true', async t => {
+  //Turning this flag on allows us to scale a sprite horizontally and vertically
+  const nativeAPI = await helpers.createDanceAPI();
+  t.ok(nativeAPI.p5_._fixedSpriteAnimationFrameSizes);
+  t.end();
+
+  nativeAPI.reset();
+});
```

Comment:
can we also have a test that makes sure that changing width/height now works?

Reply:
Before this change, the sprite would accurately save width and height as expected, but p5 wouldn't use the width and height while drawing to ensure the width/height ratio remained the same. I'll see what I can do to add a test, but it will be non-trivial to have a regression test.